assisted-events-scrape/events_scrape/__main__.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-from . import events_scrape
+import events_scrape
 
 if __name__ == '__main__':
     events_scrape.main()
```

Comment:
`Cannot find reference 'main' in '__init__.py'`
Why did you removed the relative import?